fhir-odoo-mapper/src/main/java/com/ozonehis/fhir/odoo/mappers/ChargeItemDefinitionMapper.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.fhir.odoo.mappers;
+
+import com.ozonehis.fhir.odoo.OdooConstants;
+import com.ozonehis.fhir.odoo.model.BaseOdooModel;
+import com.ozonehis.fhir.odoo.model.Currency;
+import com.ozonehis.fhir.odoo.model.ExtId;
+import com.ozonehis.fhir.odoo.model.OdooResource;
+import com.ozonehis.fhir.odoo.model.Product;
+import java.util.Map;
+import org.hl7.fhir.r4.model.ChargeItemDefinition;
+import org.hl7.fhir.r4.model.Enumerations;
+import org.hl7.fhir.r4.model.Money;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ChargeItemDefinitionMapper<O extends BaseOdooModel & OdooResource>
+        implements ToFhirMapping<O, ChargeItemDefinition> {
+
+    @Override
+    public ChargeItemDefinition toFhir(Map<String, O> resourceMap) {
+        ChargeItemDefinition chargeItemDefinition = new ChargeItemDefinition();
+        if (resourceMap == null || resourceMap.isEmpty()) {
```

Comment:
You can replace this with MapUtils.isNotEmpty is you already have commons-collections on the classpath, if not you don't have to